src/js/components/modals/AppModalComponent.jsx

```diff
@@ -147,7 +147,7 @@ var AppModalComponent = React.createClass({
     event.preventDefault();
 
     if (!Object.keys(this.state.errorIndices).length) {
-      let app = AppFormStore.app;
+      let app = AppFormStore.getApp();
```

Comment:
Yes, this is now redundant and could be remove I think.
Together with this refactoring?
https://github.com/mesosphere/marathon/issues/3148